draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,76 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
 embedded Secure Elements (eSE), or Hardware Security Modules (HSM).
 
+## Layered Attestation Procedures
+
+By definition, the Attester role takes on the duty to create Attestation Evidence.
+The fact that an Attester role is composed of several types of environments that
+can be nested or staged adds complexity to the architectural layout of how an
+Attester -- in itself -- is composed and therefore has to conduct the Claims Collection
+in order to create believable Attestation Evidence. This complexity is
+independent of the concept defined by Composite Attesters.
+Composite Attesters are inherently composed of multiple nested environments.
+The following example is intended to illustrate this composition:
```

Comment:
wording: 388 and 390 seem redundant

Reply:
Ned's diagram may fit here

Reply:
Dave had the ascii art version. Generally speaking it is easier to explain a complex concept by providing a diagram that illustrates the concept. Details about the concept can be organized according to a decomposition of the diagram. If we do this first then examples can be provided for intuition and don't need excessive explanation. We could just say "device bootstrap sequence is an example of a 'layered device' where each phase of boot implements a RATS Attester that collects claims about the next phase".

Reply:
Maybe there are other use cases that provide intuition - maybe a hypervisor loading a guest VM?

Reply:
Could we fix this together please? I find it difficult to make a editorial decision here without further input